docs/core/install/includes/linux-install-package-manager-x64-vs-arm.md

```diff
@@ -1,2 +1,3 @@
 
-_Package manager installs are only supported on the **x64** architecture_. Other architectures, such as **ARM**, must manually install the .NET Core SDK or .NET Core Runtime. For more information, see the [manually install section](#manual-install).
+> [!IMPORTANT]
+> _Package manager installs are only supported on the **x64** architecture_. Other architectures, such as **ARM**, must manually install the .NET. For more information, see the [manually install section](#manual-install).
```

Comment:
Since it's now in an "Important" alert box, the first sentence probably doesn't need to stay in italics.

"the .NET SDK" or "the .NET runtime" or ".NET"

```suggestion
> Package manager installs are only supported on the **x64** architecture. Other architectures, such as **ARM**, must manually install .NET. For more information, see the [manually install section](#manual-install).
```